src/CarbonAware.WebApi/src/Models/EmissionsForecastDTO.cs

```diff
@@ -68,7 +69,8 @@ public static EmissionsForecastDTO FromEmissionsForecast(EmissionsForecast emiss
             DataEndAt = emissionsForecast.DataEndAt,
             WindowSize = (int)emissionsForecast.WindowSize.TotalMinutes,
             OptimalDataPoint = EmissionsDataDTO.FromEmissionsData(emissionsForecast.OptimalDataPoint),
-            ForecastData = emissionsForecast.ForecastData.Select(d => EmissionsDataDTO.FromEmissionsData(d))!
+            ForecastData = emissionsForecast.ForecastData.Select(d => EmissionsDataDTO.FromEmissionsData(d))!,
+            RequestedAt = (DateTimeOffset?)(props?[CarbonAwareConstants.ForecastRequestedAt] ?? DateTimeOffset.UtcNow)
```

Comment:
hi @juzuluag, my vote would be to instead add `RequestedAt` to `EmissionsForecast` object, and to have `CarbonAwareAggregator.ProcessAndValidateForecast` be what sets the `EmmissionsForecast.RequestedAt` field, as it already does for `EmissionsForecast.DataStartAt`, `EmissionsForecase.GeneratedAt`, etc...  -- I think this follows the existing code pattern better -- thoughts?

Reply:
oh , you mean to add that property `RequestedAt`to `EmissionsForecast` object?
tbh, I don't have a strong preference. I'll make the changes.